test/intl402/NumberFormat/default-minimum-singificant-digits.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Tests that the default value of minimumSignificantDigits is 1.
+esid: sec-setnfdigitoptions
+---*/
+
+// maximumSignificantDigits needs to be in range from minimumSignificantDigits
+// to 21 (both inclusive). Setting maximumSignificantDigits to 0 will throw a
+// RangeError if the (default) minimumSignificantDigits is at least 1.
+assert.throws(RangeError, function() {
+  Intl.NumberFormat(undefined, {maximumSignificantDigits: 0});
+});
+
+// For similar reasons, the following statement is checking that
+// minimumSignificantDigits is at most 1.
+assert.notSameValue(Intl.NumberFormat(undefined, {maximumSignificantDigits: 1}), undefined);
```

Comment:
I've had something else in mind. I'm used to test something I can have from the actual result, rather than just checking it's something (not undefined), otherwise, `42` would be a false positive.

What do you think about checking the result object with the expected set values? The assertions in the example below confirms the default and given values are properly set, right?

```js
var res = Intl.NumberFormat(undefined, {maximumSignificantDigits: 1})

assert.sameValue(Object.getPrototypeOf(res), Intl.NumberFormat.prototype, 'result is an instance of NumberFormat')
assert.sameValue(res.resolvedOptions().minimumSignificantDigits, 1, 'default minimumSignificantDigits')
assert.sameValue(res.resolvedOptions().maximumSignificantDigits, 1, 'sets maximumSignificantDigits')
```